objax/functional/loss.py

```diff
@@ -48,6 +56,23 @@ def cross_entropy_logits_sparse(logits: JaxArray, labels: Union[JaxArray, int])
     return logsumexp(logits, axis=1) - logits[jn.arange(logits.shape[0]), labels]
 
 
+def mean_squared_logarithmic_error(y_true: JaxArray, y_pred: JaxArray) -> JaxArray:
+    """Computes the the mean squared logarithmic error.
+
```

Comment:
Here's a code sample of what I had in mind
```
def mean_squared_error(y_true: JaxArray, y_pred: JaxArray, keep_dims: Iterable[int] = (0,)) -> JaxArray:
    """Computes the mean squared logarithmic error between y_true and y_pred.
    
    Args:
        x: a tensor of shape (d0, .. dN-1).
        y: a tensor of shape (d0, .. dN-1).
        keep_dims: a sequence of the dimensions to keep.
        
    Returns:
        (d_i, ..., for i in keep_dims) tensor of the mean squared error.
    """
    loss = (jn.log1p(y_true) - jn.log1p(y_pred)) ** 2
    axis = [i for i in range(loss.ndim) if i not in keep_dims]
    return loss.mean(axis)
```

Using the default, it'll reduce on all axis except for 0 (which typically is the batch), which is standard in Objax. And the user can set keep_dims to their own taste if they want a different reduction.